packages/react-router-dom/server.tsx

```diff
@@ -64,7 +64,6 @@ export function StaticRouter({
 }
 
 export interface StaticRouterProviderProps {
-  basename?: string;
```

Comment:
To align with `<RouterProvider>` we no longer accept a `basename` as a prop, we inherit from the router passed to it.  In the SSR case, that further inherits from the static handler.